### ISSUE TITLE
Upload to codepen.io

### ISSUE THREAD

**Issue by 0x48piraj** [0x48piraj/chrome-incognito-page#1]

**Comment by soulspark666** [comment]
I can do it, but what do you want to do with it?

**Comment by 0x48piraj** [comment]
No, this issue is for me, for a mental note, thus not added `#hacktoberfest`

I'm going to self-assign to avoid further confusion.

**Comment by soulspark666** [comment]
Ok, still I did something:
https://codepen.io/soulspark666/full/MWgMBXj
